zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -464,7 +464,7 @@ benchmarks! {
             MarketType::Categorical(T::MaxCategories::get()),
             ScoringRule::CPMM
         )?;
-        let approval_origin = T::ApprovalOrigin::successful_origin();
+        let approval_origin = T::ApproveOrigin::successful_origin();
```

Comment:
So this works, because `ApproveOrigin` includes the origins of `RejectOrigin`. Maybe use `RejectOrigin` to avoid the need of this background information.

Reply:
You're right, we should use the respective origins. Mitigated in [d267583](https://github.com/zeitgeistpm/zeitgeist/pull/744/commits/d26758328e49a4d1ff8616557af79f9affae756b).